src/registrar/assets/sass/_theme/_usa-modal.scss

```diff
@@ -0,0 +1,7 @@
+@use "cisa_colors" as *;
+@use "uswds-core" as *;
+
+.usa-modal.wider-modal {
+    width: 40% !important;
+    max-width: 40% !important;
```

Comment:
**Question:** I see that `!important` is used for overriding, and am curious what component you're overriding + reasoning for it? (Just trying to understand, not a "this is bad!" comment)

Reply:
This is for the modal width just to increase how much wiggle room we get on that. This ended up being removed after a conversation with Rachid yesterday